lib/eclipse/tests/UnitTests.cpp

```diff
@@ -330,7 +330,7 @@ BOOST_AUTO_TEST_CASE(PVT_M_UNITS)
     BOOST_CHECK_CLOSE( pvt_m.from_si( Meas::density , 1.0 ) , 1.0 , 1.0e-10 );
     BOOST_CHECK_CLOSE( pvt_m.from_si( Meas::pressure , 1.0 ) , 9.869232667160129e-06 , 1.0e-10 );
     BOOST_CHECK_CLOSE( pvt_m.from_si( Meas::temperature_absolute , 1.0 ) , 1.0 , 1.0e-10 );
-    BOOST_CHECK_CLOSE( pvt_m.from_si( Meas::temperature , 1.0 ) , 1.0 , 1.0e-10 );
+    BOOST_CHECK_CLOSE( pvt_m.from_si( Meas::temperature , 274.15 ) , 1.0 , 1.0e-10 );
```

Comment:
Since the temperature conversion is the only conversion with a non-zero offset, could you in addition a full circle test like:
```C++
double T = 100;
BOOST_CHECK_CLOSE( metric.from_si( Meas::temperature( metric.to_si( Meas::tempeature, T)), T, 1e-10);
```

Reply:
I though about doing this myself but determined that it was unnecessary: The proposed test does a "full circle" already, but instead of doing just that it also makes sure that the "intermediate stops" are correct.

If you insist, I'll add it though.